pkg/sql/show_create_clauses.go

```diff
@@ -213,30 +213,41 @@ func formatViewQueryTypesForDisplay(
 	desc catalog.TableDescriptor,
 ) (string, error) {
 	replaceFunc := func(expr tree.Expr) (recurse bool, newExpr tree.Expr, err error) {
+		var innerExpr tree.Expr
```

Comment:
How do you feel about adding a little commentary that this dance is about resolving UDTs used in these types of expressions so that logical values and type names are formatted properly for display and that non-UDT types don't require any extra work?

Reply:
added